iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/task/builder/PipeDataNodeTaskBuilder.java

```diff
@@ -141,4 +158,37 @@ private PipeParameters blendUserAndSystemParameters(final PipeParameters userPar
     blendedParameters.putAll(systemParameters);
     return new PipeParameters(blendedParameters);
   }
+
+  private void checkConflict(
+      final PipeParameters extractorParameters, final PipeParameters connectorParameters) {
+
+    try {
+      final Pair<Boolean, Boolean> insertionDeletionListeningOptionPair =
+          DataRegionListeningFilter.parseInsertionDeletionListeningOptionPair(extractorParameters);
+      if (!insertionDeletionListeningOptionPair.left) {
+        return;
+      }
+    } catch (IllegalPathException e) {
+      LOGGER.warn(
+          "PipeDataNodeTaskBuilder failed to parse 'inclusion' and 'exclusion' parameters: {}",
+          e.getMessage());
+      return;
+    }
+
+    final Boolean isRealtime =
+        connectorParameters.getBooleanByKeys(
+            PipeConnectorConstant.CONNECTOR_REALTIME_FIRST_KEY,
+            PipeConnectorConstant.SINK_REALTIME_FIRST_KEY);
+    if (isRealtime == null) {
+      connectorParameters.addAttribute(PipeConnectorConstant.CONNECTOR_REALTIME_FIRST_KEY, "false");
+      LOGGER.warn(
+          "PipeDataNodeTaskBuilder: When 'inclusion' contains 'data.delete', 'realtime-first' is defaulted to 'false' to prevent sync issues after deletion.");
```

Comment:
This shall not print warn log because it's a normal circumstance. Either delete this or change it to info.